managers/urls.py

```diff
@@ -14,4 +14,14 @@
         views.RemoveTaskActivityView.as_view(),
         name="disable-task-activity",
     ),
+    url(
+        "^expected-hours/project/(?P<pk>[0-9]+)/(?P<year>[0-9]{4})/(?P<month>[0-9]{1,2})/$",
+        views.ProjectExpectedHoursView.as_view(),
+        name="project-expected-hours",
```

Comment:
Maybe a little more precise would be `project-expected-hours-list`?

Reply:
changed